### DISCUSSION TITLE
How to pair multiple devices?

### DISCUSSION THREAD

**Discussion by CSKoenigGbR** [HomeSpan/HomeSpan#242] [Q&A] [answered]
Hello HomeSpan!

first of all: Congratulation to this wonderful piece of technology! 🥇 

My situation:
I have three ESP32 setup with your HomeSpan library.
    1x Temp&HumiSensor
    1x HumidifierDehumidifier
    1x MainEntranceDoor
Successfully paired all devices with my iPhone 11 Pro. Every control of the devices are working like a charm. Everything perfect so far.

Now my only one problem:
I can't pair any other iPhone to any of the devices. If I try to search them in the Home-App, they are not visible. If I try to connect via QR-Code, I only get one message: "Accessory Already Added". I've tried two additional devices to verify the error is not caused by the iPhone.
Please help me! 🥲

**Comment by HomeSpan** [comment, marked as answer]
Once you pair a HomeKit device using the Home App on your iPhone it is automatically connected to your entire HomeKit network associated with your iCloud account, so it cannot be paired with another iPhone.  However, the device will automatically show up in the Home App on any other iPhone, Mac, or iPad (and can be accessed via Siri on any HomePod) provided they are associated with the same iCloud account.

To grant access to another iCloud user you need to invite them to share control of your "Home."   You'll find the menu for inviting others within the settings section of the Home App on your own iPhone.  Once others accept the invitation, your "Home" will show up on all their iPhones (Macs, iPads, etc.), and the ESP32 devices you originally paired will be accessible to them as well.    If you add additional devices in the future, they will automatically show up on your Home App as well as any other users that have shared access.  In other words, once you share your "Home" all device added in the future are automatically shared as well.
